CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.0-rc.5 (2019-01-10)
+
+- Fixed flyout component resize handle to not have extra styles in Firefox. [#15](https://github.com/blackbaud/skyux-flyout/pull/15)
```

Comment:
"flyout component resize handle" is a mouthful, and I couldn't tell what was a verb and what was a noun the first time I read this.

I'd suggest changing this to "Fixed styles on flyout's resize handle in Firefox."

Reply:
Done